modular_skyrat/modules/colony_fabricator/code/machines/solid_fuel_generator.dm

```diff
@@ -1,5 +1,5 @@
 /obj/machinery/power/port_gen/pacman/solid_fuel
-	name = "S.O.F.I.E. solid fuel generator"
+	name = "\improper S.O.F.I.E.-type portable generator"
 	desc = "The second most common generator design in the galaxy, second only to the P.A.C.M.A.N. \
```

Comment:
Still forgot to describe what S.O.F.I.E. stands for in the description here 😎

Reply:
fuark